public/lessons/javascript/the_command_line.md

```diff
@@ -40,13 +40,14 @@ See [this twitter thread](https://twitter.com/b0rk/status/980825751456047104) fo
 
 * to open your Terminal:
   * Mac OS: launch the "Terminal" application
-  * Windows: launch the "Cmdr" application (pronounced "Commander") and select "git bash" [TODO: verify on windows] 
+  * Windows: Starting in Windows Vista, there are two command shells: Windows Powershell, and Command Prompt. Launch Command Prompt (cmd).
```

Comment:
CMDR is different from CMD.EXE.  Check it out here : https://cmder.net/

Reply:
...and I misspelled it: `cmder` is the correct name

Reply:
Thanks for the shell feedback. I agree that it's confusing and overcomplicated. The multiplicity of shells on Windows is actually one reason I *like* Cmder: it separates the *shell* from the *terminal emulator*, and provides a lot of good features no matter whether you're using PowerShell, Bash, cmd.exe, or whatever. But I'm okay with keeping it optional, or recommending students use the VS Code built-in terminal if that's easy to configure.

Thanks for catching that we don't seem to have install instructions for Cmder (I think I put it in the python installfest but not the JS one). I'll correct that now.